src/main/java/mate/academy/bookstoreapp/repository/user/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.bookstoreapp.repository.user;
+
+import java.util.Optional;
+import mate.academy.bookstoreapp.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    Optional<User> findByEmail(String email);

```

Comment:
It's better to use this method to avoid calling isPresent() in Service
```suggestion
    boolean existsByEmail(String email);
```